uk.co.compucorp.civicrm.hrleaveandabsences/js/angular/src/leave-absences/shared/components/leave-widget/leave-widget.component.js

```diff
@@ -1,13 +1,29 @@
 /* eslint-env amd */
 
 define([
+  'common/angular',
   'common/lodash',
+  'common/bundles/components',
+  'common/modules/directives',
+  'common/modules/models',
+  'common/filters/time-unit-applier.filter',
+  'leave-absences/shared/modules/shared-settings',
   'leave-absences/shared/modules/components',
   'leave-absences/shared/models/absence-period.model',
   'leave-absences/shared/models/absence-type.model',
-  './leave-widget-balance.component'
-], function (_, components) {
-  components.component('leaveWidget', {
+  './leave-widget-absence-types-amount-taken.component',
```

Comment:
why the `.`

Reply:
requirejs allows you to include relative paths, which I'm using because it makes the leave widget more portable (if it needs to change places this line of code doesn't need to change).

I'm using `./component-name` instead of `component-name`, which would also work, to make it more explicit that the required component is in the same folder.

Reply:
got it, but please stick with the convention that we have for now (= use `leave-absences/`)